src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -1,7 +1,5 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-#define dark_theme
```

Comment:
Wait, we were compiling with this all along?

Reply:
Tagging @CyrusNajmabadi

Reply:
Nice! :)

Reply:
Yeah, oops.